include/layer/layer.h

```diff
@@ -0,0 +1,24 @@
+#ifndef LAYER_H
+#define LAYER_H
+
+#include <list>
+
+#include "./tensor/tensor.h"
+
+class Layer {
+ protected:
+  int id_;
+
+ public:
+  Layer() = default;
```

Comment:
please create constructor with abstract class of attributes, @aobolensk what do you think?